scripts/alignment_score.py

```diff
@@ -0,0 +1,109 @@
+from re import I
+import sys
+import os
+
+# get indexes from file paths given a delimeter, default space
+def scrapeIndexes(path: str, delim: str = " ")-> list: 
+    fileA = open(path, "r")
+    lines = fileA.readlines()
+    indexes = []
+    for line in lines:
+        indexes.append(int(
+            line.strip("\n").split(delim)[0]
+        ))
+    return indexes
+
+# calculate alignment score using needleman-wunsch algorithm given
+# 2 index files, match, mismatch, and gap penalty values
+def alignmentScore(indexA: list, indexB: list, match: int, miss: int, gap: int):

```

Comment:
the main point is creating an alignment right? not just getting an alignment score. if so maybe this function should be slightly modified?

Reply:
the function does both, it appends the score while creating the alignment

Reply:
I see, maybe rename the function to "create_alignment"?